index.js

```diff
@@ -2,27 +2,27 @@
 
 const http = require('http');
 
-
 const nullBodyError = 'Body is null';
 
 const defaultConfig = {
     host: '127.0.0.1',
     port: 9676
 };
 
-exports.setConfig = (config) => {
+function setConfig(config) {
     if (!config) {

```

Comment:
not needed

Reply:
Its needed to set the configuration before getting the string resources

Reply:
I mean, the if (!config) is not needed, because updateConfig checks if there is a config.
You could also inline this method in updateConfig, so we would have a single place to look if we need to understand how the configuration works

Reply:
ok